test-app-sitebricks.yaml

```diff
@@ -0,0 +1,148 @@
+swagger: '2.0'
+info:
+  title: 'test-app-sitebricks'
+  description: 'simple test app with sitebricks'
+  version: '1.0-SNAPSHOT'
+host: "test-app-sitebricks.appspot.com"
+schemes:
+ - https
+basePath: "/"
+paths:
+  /users:
+    get:
+      summary: "returns all users registered in the system"
+      operationId: "getUsers"
+      produces:
+      - "application/xml"
+      - "application/json"
+      responses:
+        200:
+          description: "successfully return all users"
+    post:
+      summary: "register user in the system"
+      operationId: "registerUser"
+      consumes:
+      - "application/xml"
+      - "application/json"
+      produces:
+      - "application/xml"
+      - "application/json"
+      parameters:
```

Comment:
your parameters definition is invalid (use http://editor.swagger.io/#!/  to check your definitions)